compiler/AST/primitive.cpp

```diff
@@ -391,6 +391,8 @@ void
 initPrimitive() {
   primitives[PRIM_UNKNOWN] = NULL;
 
+
+  prim_def(PRIM_GET_COMPILER_VAR, "get compiler variable", returnInfoStringC);
```

Comment:
Really minor, but is there a more logical place to put this? 

We try to group primitives into logical sections, though I don't see an obvious place for this one. Maybe somewhere around `_get_user_file` or possibly `chpl_lookupFilename`?